index.html

```diff
@@ -181,5 +182,45 @@ <h3 class="maryPoppins">Skills</h3>
             </section>
         </div>
     </header>
+
+    <section class="getstarted decoration1">
+        <header>
+            <h2>Get started</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+        </header>
+        <form action="https://formspree.io/f/myyvzejv" method="post">
+            <ul>
+                <li class="posrev">
+                    <label for="name" class="crazylabel">Your name</label>
+                    <input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+                </li>
+                <li class="posrev">
+                    <label for="email" class="crazylabel">Your email</label>
+                    <input type="email" id="email" name="user_email" placeholder="Email address" required>
+                </li>
+                <li>
+                    <textarea name="user_message" maxlength="500" required>Enter your message here...</textarea>
```

Comment:
- [ ] Kindly be noticed that the `textarea` tag should have a `placeholder` attribute, also you have to remove the text in the `textarea` tag, and the `textarea` should be like this.
```html
<textarea name="user_message" maxlength="500" placeholder="Enter your message here..." required></textarea>
```